styles/figures.css

```diff
@@ -5,9 +5,13 @@ figure.fig-quote figcaption > p:before {
   content: '—';
   padding-right: 0.5em;
 }
-figure.fig-quote.pull-quote > blockquote { 
+figure.fig-quote.pull-quote > blockquote {
   font-size: 1.4rem;
 }
+/* Center captions for standard figures to match centered images */
+figure:not(.fig-quote) figcaption {
```

Comment:
Just to make sure I understand: is this CSS rule sufficient to capture all cases?

Reply:
OK I figured out a more targeted rule to just capture the `figure` directive case, because I realized that we're using `<figure>` in a few other ways as well (like pull quotes)